scripts/building_placer.gd

```diff
@@ -3,19 +3,14 @@ extends Node3D
 @export var max_number_buildings_on_x_axis : int = 20
 @export var max_number_buildings_on_z_axis : int = 20
 @export var distance_between_buildings_metres : int = 20
-@export var inherited_offset_correction : int = -220
-
-# Toggle to stop the physics process fuction repeating the building placement task
-var rays_fired : int = 0
+@export var inherited_offset_correction : int = 0
 
 var building001 = preload("res://Scenes/Buildings/Building001.tscn")
 
-# Called when the node enters the scene tree for the first time.
 func _ready():
 	pass
```

Comment:
As you're not using it you should remove this function entirely. Understanding the code takes fewer cognitive calories when there's less there to read, and I'm not sure but it might be a minor performance optimisation too - depending on how the gdscript is handled under the hood.